index.html

```diff
@@ -0,0 +1,202 @@
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+            </div>
+            <div class="work-details-row3">
+                <div class="wdr31">
+                    <h4>Profesional Art Printing Data More</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+
+                <div class="wdr32">
+                    <h4>Data Dashboard?Healthcare</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+                <div class="wdr33">
+                    <h4>Website Protfolio</h4>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard
+                    </p>
+                    <div class="sp1"><span>HTML</span><span>CSS</span><span>Javascript</span></div>
+                </div>
+            </div>
+
+
+        </div>
+
+    </div>
+    <!------------------about me------------------------------------------------>
+    <div class="aboutme-container">
+        <div class="aboutme-row1">
+            <div class="aboutme-left">
+                <h4>About me</h4>
+                <p>Hey there! I can help you build a product, feature or website. Look through some of my work and
+                    experience! If you like
+                    what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <p class="gr">Get My Resume</p>
+            </div>
+            <div class="aboutme-right">
+                <img src="./images/aboutme.png">
+            </div>
+
+        </div>
+        <div class="hline">
+            <hr>
+        </div>
+        <div class="aboutme-row2">
+            <div class="aboutme-row2-left">
+                <h3><img src="./images/Rectangle 55.png">Language</h3>
+                <h6>c</h6>
+                <h6>C++</h6>
+                <h6>Javascript</h6>
+            </div>
+            <div class="aboutme-row2-middle">
+                <h3><img src="./images/Rectangle 56.png">Frame works</h3>
+                <h6>React</h6>
+                <h6>django</h6>
+                <h6>Ruby on Rails</h6>
+            </div>
+            <div class="aboutme-row2-right">
+                <h3><img src="./images/Rectangle 57.png">Skills</h3>
+                <h6>Github</h6>
+                <h6>Web Development</h6>
+                <h6>Coding</h6>
+            </div>
```

Comment:
- [ ] You did a good job :+1: but it is required to each technology category ("Languages", "Frameworks" and "Skills") to be in a `<ul>` and each of them in a `<li>`. Also, The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should be a list `<ul>` and each of them a `<li>`